dev/src/External/CashierDesk.java

```diff
@@ -96,6 +96,8 @@ private void displayStores() {
             case StoreConstants.DISPLAY_ORDERS_INDEX:
                 displayOrders();
                 break;
+            case StoreConstants.RETURN_TO_MAIN_MENU_INDEX:
+                break;
```

Comment:
שולי מאוד אבל אולי return בשביל קריאות, לא באמת ביג דיל

Reply:
Fixed in #50